### ISSUE TITLE
Not able to add additional wzye vacuum - configuration not loading

### ISSUE THREAD

**Issue by dwdonnelly** [romedtino/simple-wyze-vac#81]
**Describe the bug**
I am trying to add an additional wyze vacuum to the integration.

Reviewed documentation located here: https://github.com/romedtino/simple-wyze-vac/blob/main/README.md
Could see no guidance on how to add additional wyze vacuum device.

I tried to "add device"...in the screen integration.  Asked for wyze credentials  - entered credentialls, stated wyze was already configured.

The following is from the log:
Logger: homeassistant.config_entries
Source: config_entries.py:586
First occurred: 2:38:39 PM (4 occurrences)
Last logged: 2:45:17 PM

Error setting up entry Simple Wyze Vac for simple_wyze_vac
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 586, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/simple_wyze_vac/__init__.py", line 95, in async_setup_entry
    "filter": vac_info.supplies.filter.remaining,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/wyze_sdk/models/devices/vacuums.py", line 970, in remaining
    return self.type.max_hours - self.usage
           ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
TypeError: unsupported operand type(s) for -: 'int' and 'NoneType'


**Simple Wyze Vac Version**
1.9.0

**Wyze Vacuum Firmware Version**
1.6.306

**Home Assistant Version**
Core   2024.7.3
Supervisor   2024.06.2
Operating System   12.4
Frontend   20240710.0

Your help is appreciated.

**Comment by romedtino** [comment]
1.6.306 is currently unsupported until wyze-sdk supports it. Shaun probably needs access to a vacuum with this version so he can figure out the correct endpoints for that verison

 https://github.com/romedtino/simple-wyze-vac/issues/75
https://github.com/shauntarves/wyze-sdk/issues/130

**Comment by dwdonnelly** [comment]
@romedtino Great - so I have a new vacuum (firmware) that is not yet supported in the integration.  Just my luck.  OK - good info and I appreciate the feedback (very helpful)!  Obviously, the next question is: any idea when it will be supported?

Also - as I was not able to add this new vacuum, I still have the problem of the integration failing to set up, the log entries provided above.

What steps should be taken to resolve this issue?  Standing by.

**Comment by dwdonnelly** [comment]
@romedtino Before I jump and do something strange, should I remove and reload the simple wyze vac integration - or do you have a better suggestion?  Thanks in advance for your help.

**Comment by Jizzay1** [comment]
I have two vacuums, one at 1.6.306 and one at 1.6.202.  The integration accepts the login and keys, but produces the same error as the original poster, and neither vacuum loads. Happy to help if i can

**Comment by bballfanatic365** [comment]
@romedtino unfortunately I'm having the same issue - see below for my logs.

Logger: homeassistant.config_entries
Source: config_entries.py:635
First occurred: 2:35:53 PM (3 occurrences)
Last logged: 2:39:37 PM

Error setting up entry Simple Wyze Vac for simple_wyze_vac
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 635, in __async_setup_with_context
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/simple_wyze_vac/__init__.py", line 95, in async_setup_entry
    "filter": vac_info.supplies.filter.remaining,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/wyze_sdk/models/devices/vacuums.py", line 970, in remaining
    return self.type.max_hours - self.usage
           ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
TypeError: unsupported operand type(s) for -: 'int' and 'NoneType'

Simple Wyze Vac Version
1.9.1

unfortunately I have no idea what's causing this, is it something that you can look into? Happy to provide any other information if needed.

**Comment by dwdonnelly** [comment]
@Jizzay1 and @bballfanatic365 Well, I purchased and tried to load another vacuum.  Same problem.  
However.....

After selecting the reload integration a dozen times over several days and restarting HA a few times....and having no success adding in the additional vacuum, I decided to pause my effort for a while.  Then today, I was looking through my integrations and - whamo!  The Simplae Wyze Vac integration had reloaded and accepted the 4th vacuum.  The only thing I had done was update a few integrations: Chimes, etc.  To the best of my knowledge (which is limited), I did not impact the Simple Wyze Vac integration.

I'm not sure what happened, but the new vacuum is there and operational.

Just thought I would pass along my experience.  Hope this helps.
